sdk/python/lib/pulumi/runtime/resource.py

```diff
@@ -607,16 +718,14 @@ async def do_register():
                 additionalSecretOutputs=additional_secret_outputs,
                 importId=opts.import_,
                 customTimeouts=custom_timeouts,
-                aliases=[alias_pb2.Alias(urn=alias) for alias in resolver.aliases],
```

Comment:
Ah this isn't right, old engines (i.e. monitor_supports_alias_specs = false) need aliasURNs sent, not aliases.

Reply:
I was confused here too because it was `[alias_pb2.Alias(urn=alias) for alias in resolver.aliases]` before, not sending the `aliasURNs` 🤔 maybe no one was hitting the problem

Reply:
Fixed it ✅